roles/cockpit/meta/main.yml

```diff
@@ -12,7 +12,4 @@ galaxy_info:
   categories:
   - cloud
 dependencies:
-- role: os_firewall
-  os_firewall_allow:
-  - service: cockpit-ws
-    port: 9090/tcp
+- role: lib_os_firewall
```

Comment:
Stating once here:
I'd rather we move to having a single role with our modules than having each in its own lib_<module_name> role.  This is not an issue to hold up this PR, just something to consider.